LiskHQ/lisk-mobile
# Change splash screen - Closes #869

# What was the bug or feature?
Described in #869.

## How did you fix it?
- Added splash screen to iOS and Android.
- Updated quick action icons on Android.
- Updated app icons on Android.

TODO:
- [x] Update icons mipmap for Android with new background.
- [x] Test on Android



## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
- Make sure splash screen and initial animation are in line with design in both iOS and Android.
- Make sure app and quick action icons are correct on Android.


# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
